Classes/Scheduler/Base.php

```diff
@@ -171,9 +171,15 @@ public static function initFrontend( $pageid )
 		$GLOBALS['TSFE']->initTemplate();
 		$GLOBALS['TSFE']->getConfigArray();
 
-		$rootline = BackendUtility::BEgetRootLine( $pageid );
-		if( ( $_SERVER['HTTP_HOST'] = BackendUtility::firstDomainRecord( $rootline ) ) == null ) {
-			throw new \RuntimeException( 'No domain record in root page' );
+		$rootline = BackendUtility::BEgetRootLine($pageid);
+		if ( BackendUtility::firstDomainRecord($rootline) != null ) {
+			$_SERVER['HTTP_HOST'] = BackendUtility::firstDomainRecord($rootline);
+		} elseif ( isset($GLOBALS['TYPO3_CONF_VARS']['EXTCONF']['aimeos']['scheduler_domain']) ) {
+			$_SERVER['HTTP_HOST'] = $GLOBALS['TYPO3_CONF_VARS']['EXTCONF']['aimeos']['scheduler_domain'];
```

Comment:
EXTCONF is a bad idea for storing such data

Reply:
Yes, it's not the best option